app/components/forms/wizard/basic-details-step.js

```diff
@@ -142,7 +142,8 @@ export default Component.extend(FormMixin, EventWizardMixin, {
       return parseInt($('.ui.form').form('get value', 'ticket_min_order'), 10) <= parseInt($('.ui.form').form('get value', 'ticket_max_order'), 10);
     };
     $.fn.form.settings.rules.checkValidTimeDifference = () => {
-      return moment($('[name=start_time]')[0].value, 'HH:mm').isBefore(moment($('[name=end_time]')[0].value, 'HH:mm'));
+      if ($('[name=start_date]')[0].value === $('[name=end_date]')[0].value) {return moment($('[name=start_time]')[0].value, 'HH:mm').isBefore(moment($('[name=end_time]')[0].value, 'HH:mm'))}
```

Comment:
Use &&

Reply:
sir how can i use && in this logic $('[name=start_date]')[0].value === $('[name=end_date]')[0].value

Reply:
That's not what I said. There is no reason to have multiple returns here